external/warc/src/main/java/com/digitalpebble/stormcrawler/warc/GzipHdfsBolt.java

```diff
@@ -15,30 +15,21 @@
 package com.digitalpebble.stormcrawler.warc;
 
 import java.io.IOException;
-import java.net.URI;
 import java.util.ArrayList;
 import java.util.List;
-import java.util.Map;
 import org.apache.hadoop.fs.FSDataOutputStream;
-import org.apache.hadoop.fs.FileSystem;
 import org.apache.hadoop.fs.Path;
-import org.apache.storm.hdfs.bolt.AbstractHdfsBolt;
-import org.apache.storm.hdfs.bolt.format.FileNameFormat;
+import org.apache.storm.hdfs.bolt.HdfsBolt;
 import org.apache.storm.hdfs.bolt.format.RecordFormat;
-import org.apache.storm.hdfs.bolt.rotation.FileRotationPolicy;
-import org.apache.storm.hdfs.bolt.sync.SyncPolicy;
 import org.apache.storm.hdfs.common.AbstractHDFSWriter;
 import org.apache.storm.hdfs.common.HDFSWriter;
-import org.apache.storm.hdfs.common.rotation.RotationAction;
-import org.apache.storm.task.OutputCollector;
-import org.apache.storm.task.TopologyContext;
 import org.apache.storm.tuple.Tuple;
 import org.apache.storm.utils.Utils;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 /** Unlike the standard HdfsBolt this one writes to a gzipped stream with per-record compression. */
-public class GzipHdfsBolt extends AbstractHdfsBolt {
+public class GzipHdfsBolt extends HdfsBolt {
```

Comment:
There must have been a reason why we chose to extend  _AbstractHdfsBolt_ but can't remember what it could have been. Or maybe there wasn't? It has been a while...